# should also add which camera the reference frame is based
#ArucoPose[] arucos
int32[] ids
geometry_msgs/Pose[] poses